.gitbook.yaml

```diff
@@ -0,0 +1,5 @@
+root: ./
+
+structure:
+  readme: INTRODUCTION.md
```

Comment:
Why are these two files all upper case?

Reply:
GitBook generates these 2 main files in uppercase. No other reason. With .gitbook.yaml we can probably rename them to lowercase.

Reply:
You may have some troubles lower casing the actual files, if you use osx.

Reply:
Ran into this already 😅 , had to:

```
$ git mv somename tmpname
$ git mv tmpname SomeName
```